meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2018.01.inc

```diff
@@ -0,0 +1,23 @@
+HOMEPAGE = "http://www.denx.de/wiki/U-Boot/WebHome"
+SECTION = "bootloaders"
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=a2c678cfd4a4d97135585cad908541c6"
+PE = "1"
+
+# We use the revision in order to avoid having to fetch it from the
+# repo during parse
+
+# andy-shev's repo provides both u-boot with and without acpi enabled 
+#tag acpi (acpi enabled)
+SRCREV = "f5d7d4a7027fae6667fd4224cdf24fdc87c8f5f0"
```

Comment:
This and the non-acpi (commented out) revisions don't seem to work for me, bitbake fails while fetching and I also can't find thos manually in the repo.

Reply:
Ok, the solution is to remove the `branch=acpi` from the `SRC_URI`. That's actually surplus, because commit hashes are unique across branches anyway.

Reply:
Replacing ones in the recipe with the following ones builds ok. Not sure if those match the ones you had in mind, worth checking:
1. ACPI version (tip of the `acpi` branch): https://github.com/andy-shev/u-boot/commit/1e974c4d2b3a6ee292c8ff22ffe4994efc1fdf23
1. non-ACPI version (the last commit before enabling ACPI in the config): https://github.com/andy-shev/u-boot/commit/21ba23b0a522f7898afe1316d9800b0f7404bcc6

Reply:
This is weird. Maybe bitbake found the wrong (old) sha in my local cache and then did not bother to fetch? Will check. So in the future before I push this stuff I should always test after doing `make update`. Or I did something else dumb.

The sha's you found are correct and 21ba23b is also found in the acpi branch.

Reply:
An alternative could be a rebase + forced push by Andy, that changes commit hashes, but namely because of that is not recommended and is considered a bad practice, so I doubt Andy would do that.

Reply:
Maybe. Let me check.

Reply:
Btw, for non-acpi U-Boot, please, modify recipe to use U-Boot upstream (v2018.01+) + patches (if needed).

Reply:
OK

Reply:
I see. I copies the sha's from my local repo instead of from remote. Then since bitbake finds the sha in the local copy it does not fetch, so remote rebase remains undetected. My fault. Next time I'll remember to copy the sha from github.